openslides/core/static/js/core/site.js

```diff
@@ -1311,25 +1323,19 @@ angular.module('OpenSlidesApp.core.site', [
             if ($scope.formFields[i].key == "showAsAgendaItem") {
                 // get state from agenda item (hidden/internal or agenda item)
                 $scope.formFields[i].defaultValue = !customslide.agenda_item.is_hidden;
+            } else if($scope.formFields[i].key == "agenda_parent_item_id") {
+                $scope.formFields[i].defaultValue = customslide.agenda_item.parent_id;
             }
         }
 
         // save form
         $scope.save = function (customslide) {
-            // inject the changed customslide (copy) object back into DS store
-            Customslide.inject(customslide);
-            // save change customslide object on server
-            Customslide.save(customslide).then(
+            Customslide.create(customslide).then(
                 function(success) {
-                    // save agenda specific stuff
-                    var type = customslide.showAsAgendaItem ? 1 : 2;
-                    if (customslide.agenda_item.type != type) {
-                        customslide.agenda_item.type = type;
-                        Agenda.save(customslide.agenda_item);
-                    }
-                    $scope.closeThisDialog();
-                },
-                function (error) {
+                    var changes = [{key: 'type', value: (customslide.showAsAgendaItem ? 1 : 2)},
+                                   {key: 'parent_id', value: customslide.agenda_parent_item_id}];
+                    AgendaUpdate.saveChanges(success.agenda_item_id,changes);
+                }, function (error) {
```

Comment:
You have to add `$scope.closeThisDialog();` before this line - otherwise the customslide update form is not closed after saving...